src/core/infrastructure/log.d.ts

```diff
@@ -1,6 +1,9 @@
 export declare class Log {
 	constructor();
+
```

Comment:
Can we remove this and also add eslint rule no-useless-constructor?

Reply:
no to write constructor is a good style even empty

Reply:
@klumba12 ok 😸